wiki/Tournaments/MCNC/2020/en.md

```diff
@@ -0,0 +1,404 @@
+  - [ChunbaiP - Guang (\[Crz\]xz1z1z) \[Charoi's Medium 1.1x\]](https://osu.ppy.sh/b/2436144)
+  - [Denpagumi.inc - Den Den Passion (frolica) \[x1.1\]](https://osu.ppy.sh/b/2235083)
+  - [DJ Myosuke - Anguish (Guilhermeziat) \[Calamity\]](https://osu.ppy.sh/b/1461055)
+  - [FELT - OUR SHIP (Imperial Wolf) \[Ark\]](https://osu.ppy.sh/b/2426038)
+  - [Zekk - Fluctuation (Urusai) \[Stage 4: Variation\]](https://osu.ppy.sh/b/2423454)
+  - [Paul Bazooka - Drunken Stein (Sillyp) \[KUNNAN BIANSU\]](https://osu.ppy.sh/b/2429361)
+  - [PSYQUI feat. Mikanzil - Endroll (Macan0133) \[x1.0\]](https://osu.ppy.sh/b/2353780)
+  - [SHIKI - Angelic layer (Madoka2574) \[sc\]](https://osu.ppy.sh/b/2332060)
+  - [Sobrem feat.Sound piercer - HoneyButterBass (Sillyp) \[4k Extra\]](https://osu.ppy.sh/b/2436739)
+  - [VerseQuence - Wilt (Guilhermeziat) \[Unweave\]](https://osu.ppy.sh/b/1253611)
+  - [ZYTOKINE - DESIRE DREAM feat. Itori - FELT Remix (\_underjoy) \[Melancholia\]](https://osu.ppy.sh/b/1751934)
+- Tiebreaker
+  - **[Nizikawa ft. Nayuta - Babel (\_VianK\_) \[Birdcage\]](https://osu.ppy.sh/b/2190865)**
+
+### Group Stage
+
+**[Download the mappack here! (121 MB)](https://www.dropbox.com/s/lqde07vuqftq579/MCNC20%20%E5%B0%8F%E7%BB%84%E8%B5%9B%E5%9B%BE%E5%8C%85.rar?dl=0)**
+
+- FreeMod
+  - [Bring Me The Horizon - ouch (Sillyp) \[Tears\]](https://osu.ppy.sh/b/2413394)
+  - [ck - Carnation (ck remix) (Tamaki Iroha) \[Cyaegha\]](https://osu.ppy.sh/b/2333548)
+  - [Bullet For My Valentine - Waking The Demon (Shoegazer) \[Expert\]](https://osu.ppy.sh/b/2159735)
+  - [Puru - Filament (Sillyp) \[Insane\]](https://osu.ppy.sh/b/2242164)
+  - [Various - SN3RM Collection - 4K LN (Minseo2000) \[Killerblood - Theme of Kingdom No. 8\]](https://osu.ppy.sh/b/1399606)
+  - [Dreamcatcher - Endless Night (LovelyN) \[Finite\]](https://osu.ppy.sh/b/2340614)
+  - [SUWAKI - Shungiku (-mint-) \[Chrysanthemum\]](https://osu.ppy.sh/b/2293711)
+  - [Hong Liu Zhi Yuan - Mouse Wizard (\[RT\]Alleyne) \[Mouse\]](https://osu.ppy.sh/b/2126573)
+  - [Hommarju - BEAST BASS BOMB (-MysticEyes) \[MAXIMUM\]](https://osu.ppy.sh/b/1940816)
+  - [Nanawo Akari - DaDaDaDa Tenshi (Xingyue-) \[Dame\]](https://osu.ppy.sh/b/2352772)
+  - [ITSUHA. - Klouz of Reveria (Elekton) \[cascade\]](https://osu.ppy.sh/b/1514656)
+- Tiebreaker
+  - **[Camellia - Tornado (Dubstek) \[Disaster\]](https://osu.ppy.sh/b/2021759)**
+
+### Qualifiers
+
+**[Download the mappack here! (14.3 MB)](https://www.dropbox.com/s/xwvugt6bvhy6d20/MCNC20%20%E8%B5%84%E6%A0%BC%E8%B5%9B%E5%9B%BE%E5%8C%85.rar?dl=0)**
+
+- NoMod
+  - [DASU - 8 6 (\[Crz\]xz1z1z) \[SV\]](https://osu.ppy.sh/b/2285630)
+  - [Falk - Mach Sigma - For Dream Dance Act 1 (shuniki) \[ShuChan!!\]](https://osu.ppy.sh/b/1996937)
+  - [AKINO from bless4 - Miiro (Mat) \[Acc Meme\]](https://osu.ppy.sh/b/1759637)
+  - [KBSM - Blue Rose (LegendXYZ) \[Hard\]](https://osu.ppy.sh/b/237790)
+  
+## Match results
+
```

Comment:
`*Win by Default*` -> `*win by default*`

Reply:
*win by default* changed
but i dont think we need to use 0 | -1 when MWC use MAX | 0